bitnami/rabbitmq-cluster-operator/templates/messaging-topology-operator/validating-webhook-configuration.yaml

```diff
@@ -9,7 +9,7 @@ SPDX-License-Identifier: APACHE-2.0
 {{- $secretName := printf "%s" (include "rmqco.msgTopologyOperator.webhook.fullname" .) }}
 {{- $ca := genCA "rmq-msg-topology-ca" 365 }}
 {{- $cert := genSignedCert (include "rmqco.msgTopologyOperator.fullname" .) nil (list (printf "%s.%s.svc" (include "rmqco.msgTopologyOperator.webhook.fullname" .) (include "common.names.namespace" .)) (printf "%s.%s.svc.%s" (include "rmqco.msgTopologyOperator.webhook.fullname" .) (include "common.names.namespace" .) .Values.clusterDomain)) 365 $ca }}
-{{- if and (not .Values.useCertManager) (not .Values.msgTopologyOperator.existingWebhookCertSecret) }}
+{{- if and (.Values.msgTopologyOperator.enabled) (not .Values.useCertManager) (not .Values.msgTopologyOperator.existingWebhookCertSecret) }}
```

Comment:
Surround the whole file with an outer `{{ if .Values.msgTopologyOperator.enabled }}` condition in order to avoid this file being generated with `---` as its unique content (because line 31 is currently out of the scope of the conditionals)